src/CliMenuBuilder.php

```diff
@@ -198,18 +198,36 @@ public function setExitButtonText(string $exitButtonText) : self
         return $this;
     }
 
-    public function setBackgroundColour(string $colour) : self
-    {
-        Assertion::inArray($colour, MenuStyle::getAvailableColours());
+    public function setBackgroundColour($colour, string $fallback = null) : self
+    {
+        if (is_int($colour)) {
+            if ($this->terminal->getColourSupport() < 256) {
```

Comment:
Looks like we can extract this validation as it's the same in `setForegroundColour`. Lets introduce a `validateColour` method.